packages/react-strict-dom/tests/css-test.native.js

```diff
@@ -1066,10 +1066,10 @@ describe('properties: custom property', () => {
         'color',
         'var(--testUnfinished'
       ])
-    ).toEqual('var(--testUnfinished');
-    expect(
+    ).toEqual(undefined);
+    expect(() =>
       resolveCustomPropertyValue(customProperties, ['color', 'var(bad--input)'])
-    ).toEqual('var(bad--input)');
+    ).toThrow(new TypeError('Invalid custom property name: bad--input'));
```

Comment:
Not sure we should be throwing errors here as on web this wouldn't crash the app

Reply:
IMO, we should warn and ignore this value to match the behaviour on the web.

Reply:
Agreed — will make this change